pkg/snapstore/utils.go

```diff
@@ -46,24 +46,6 @@ func GetSnapstore(config *brtypes.SnapstoreConfig) (brtypes.SnapStore, error) {
 		return nil, fmt.Errorf("storage container name not specified")
 	}
 
-	if len(config.TempDir) == 0 {
-		config.TempDir = path.Join("/tmp")
-	}
-	if _, err := os.Stat(config.TempDir); err != nil {
-		if !os.IsNotExist(err) {
-			return nil, fmt.Errorf("failed to get file info of temporary directory %s: %v", config.TempDir, err)
-		}
-
-		logrus.Infof("Temporary directory %s does not exist. Creating it...", config.TempDir)
-		if err := os.MkdirAll(config.TempDir, 0700); err != nil {
-			return nil, fmt.Errorf("failed to create temporary directory %s: %v", config.TempDir, err)
-		}
-	}
-
-	if config.MaxParallelChunkUploads <= 0 {
-		config.MaxParallelChunkUploads = 5
-	}
```

Comment:
any reason for removing this check ?

Reply:
We already validate this field here - https://github.com/gardener/etcd-backup-restore/blob/2fcbadd7e6d57b302e8ae3b9c0913bd539b1b693/pkg/types/snapstore.go#L225-L227

So there's no point in checking again and setting to a default value, because it's already validated during initialization of the object.